utils/src/ooinstall/cli_installer.py

```diff
@@ -773,23 +794,23 @@ def cli(ctx, unattended, configuration, ansible_playbook_directory, ansible_conf
 
     # If no playbook dir on the CLI, check the config:
     if not ansible_playbook_directory:
-        ansible_playbook_directory = oo_cfg.settings.get('ansible_playbook_directory', '')
+        ansible_playbook_directory = oo_cfg.deployment.variables.get('ansible_playbook_directory', '')
```

Comment:
ansible_playbook_directory should be at the root of the config rather than under deployment.